sktime/registry/_base_classes.py

```diff
@@ -77,6 +81,16 @@
     ("regressor", BaseRegressor, "time series regressor"),
     ("forecaster", BaseForecaster, "forecaster"),
     ("transformer", BaseTransformer, "time series transformer"),
+    (
+        "transformer-pairwise-tab",
+        BasePairwiseTransformer,
+        "pairwise transformer for tabular data, distance or kernel",
+    ),
+    (
+        "transformer-pairwise-ts",
+        BasePairwiseTransformerPanel,
+        "pairwise transformer for tabular data, distance or kernel",
```

Comment:
Update this to "pairwise transformer for panel data, distance or kernel"

Reply:
fixed